src/main/java/edu/harvard/iq/dataverse/api/Admin.java

```diff
@@ -1035,15 +1035,33 @@ public Response toggleSuperuser(@PathParam("identifier") String identifier) {
 		ActionLogRecord alr = new ActionLogRecord(ActionLogRecord.ActionType.Admin, "toggleSuperuser")
 				.setInfo(identifier);
 		try {
```

Comment:
GitHub doesn't let me leave this comment on the right line but I think we should mark the old toggleSuperuser method as deprecated (in the guides and release note too).

Reply:
@pdurbin done